apps/judicial-system/backend/src/app/modules/case/guards/rolesRules.ts

```diff
@@ -1,5 +1,6 @@
 import { RolesRule, RulesType } from '@island.is/judicial-system/auth'
 import {
+  CaseIndictmentRulingDecision,
```

Comment:
Is this a mistake? Did you add this and never use it?

Reply:
Yep mistake! Removed 😊